### DISCUSSION TITLE
FreeBSD supported?

### DISCUSSION THREAD

**Discussion by gdonald** [mason-org/mason.nvim#1011] [Q&A]
I'm not sure if this is the correct place to report this issue.  Please let me know where to go if not here.  Thanks.

When I start nvim after cloning https://github.com/nvim-lua/kickstart.nvim into my ~/.config/nvim I get this error:

[ERROR Sun Feb 19 09:02:06 2023] ...e/nvim/lazy/mason.nvim/lua/mason-core/installer/init.lua:192: Installation failed for Package(name=lua-language-server) error="Could not find which release file to download.\nMost likely the current operating system or architecture is not supported (FreeBSD_amd64)."

**Comment by freddylist** [comment]
This would be nice, but Mason seems to favor downloading binaries of which probably none are for FreeBSD. :'(

We could maybe patch Mason to download the Linux binaries anyway and use the Linuxulator to run them, but that seems kinda jank considering much in the Mason package registry is also packaged by FreeBSD (including [lua-language-server](https://www.freshports.org/devel/lua-language-server/)).

Maybe a slightly less jank could be to try and get [Nix](https://nixos.org/) to work for us here, but it is experimental on FreeBSD.
